icon.svg

```diff
@@ -1,139 +1 @@
-         style="fill:#ffffff;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:0.32031175"
-         id="path92-5"
-         inkscape:connector-curvature="0" />
-    </g>
-    <g
-       id="g94-6"
-       transform="matrix(4.162611,0,0,-4.162611,367.36686,631.05679)"
-       style="stroke-width:0.32031175">
-      <path
-         d="m 0,0 c 0,-7.994 -6.479,-14.473 -14.479,-14.473 -7.996,0 -14.479,6.479 -14.479,14.473 0,7.994 6.483,14.479 14.479,14.479 C -6.479,14.479 0,7.994 0,0"
-         style="fill:#414042;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:0.32031175"
-         id="path96-2"
-         inkscape:connector-curvature="0" />
-    </g>
-    <g
-       id="g98-9"
-       transform="matrix(4.162611,0,0,-4.162611,511.99336,724.73954)"
-       style="stroke-width:0.32031175">
-      <path
-         d="m 0,0 c -3.878,0 -7.021,2.858 -7.021,6.381 v 20.081 c 0,3.52 3.143,6.381 7.021,6.381 3.878,0 7.028,-2.861 7.028,-6.381 V 6.381 C 7.028,2.858 3.878,0 0,0"
-         style="fill:#ffffff;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:0.32031175"
-         id="path100-1"
-         inkscape:connector-curvature="0" />
-    </g>
-    <g
-       id="g102-2"
-       transform="matrix(4.162611,0,0,-4.162611,634.78706,625.67104)"
-       style="stroke-width:0.32031175">
-      <path
-         d="m 0,0 c 0,-12.052 9.765,-21.815 21.815,-21.815 12.041,0 21.808,9.763 21.808,21.815 0,12.044 -9.767,21.802 -21.808,21.802 C 9.765,21.802 0,12.044 0,0"
-         style="fill:#ffffff;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:0.32031175"
-         id="path104-7"
-         inkscape:connector-curvature="0" />
-    </g>
-    <g
-       id="g106-0"
-       transform="matrix(4.162611,0,0,-4.162611,656.64056,631.05679)"
-       style="stroke-width:0.32031175">
-      <path
-         d="m 0,0 c 0,-7.994 6.477,-14.473 14.471,-14.473 8.002,0 14.479,6.479 14.479,14.473 0,7.994 -6.477,14.479 -14.479,14.479 C 6.477,14.479 0,7.994 0,0"
-         style="fill:#414042;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:0.32031175"
-         id="path108-9"
-         inkscape:connector-curvature="0" />
-    </g>
-  </g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 999 999"><g fill="#fff"><path d="M102 657v32q397 345 795 0v-32z"/><path fill="#478cbf" d="M128 232q-46 48-81 106 26 41 55 79v240l149 14q12 2 14 15l4 64 128 10 9-60q2-11 14-14H579q12 3 14 15l8 59 129-9 4-67q4-12 14-13l149-14V417q29-38 55-79-35-57-81-106-42 21-80 46-39-35-86-62 6-50 7-100-56-27-120-41-25 41-45 87-48-7-95 0-19-44-45-87-62 13-120 41 1 50 7 100-46 26-86 62-39-26-80-46M102 689v39c3 272 798 267 795 0v-39l-132 12-5 67q-1 10-13 13l-157 11q-12 1-17-10l-10-64H436l-10 63q-3 10-13 11l-161-11q-10-2-13-12l-5-68Z"/><path d="M471 502v84c3 33 54 33 57 0v-84c-3-32-53-33-57 0"/><circle cx="708" cy="514" r="88"/><circle cx="291" cy="514" r="88"/></g><g fill="#414042"><circle cx="300" cy="520" r="59"/><circle cx="700" cy="520" r="59"/></g></svg>
```

Comment:
I think our SVG library works better if you have fixed width and height in the `<svg>` tag, not just viewBox.